test/var/log/README.md

```diff
@@ -0,0 +1,7 @@
+The `parallel_runtime_rspec.*.log` files in this directory are here on purpose.
```

Comment:
Oof, splitting them up manually like this makes sense as a work-around, but this is something that should be handled by the testing framework with a queue / workers 😢

I've been considering something similar to help get Node CI times down

Reply:
Just to be clear, I'm expressing frustration at this problem myself. This is not saying you shouldn't do this

Reply:
Yea. RSpec :( You don't even need external queues or workers really for Hatchet, as it's 99% I/O wait. If `parallel_tests` didn't pre-assign spec files to the processes it spawned, but instead let them work one by one, none of this would even be necessary.